### ISSUE TITLE
Issue 0.14.0 / 0.14.1 type_id out of bounds in get_index(8.3.*)

### ISSUE THREAD

**Issue by abitmore** [steemit/steem#408]
The function `follow_evaluator::do_apply` shouldn't be called by a node without follow_plugin enabled.

```
409822ms th_a       follow_evaluators.cpp:149     do_apply             ] 10 assert_exception: Assert Exception
_index[space_id].size() > type_id: type_id out of bounds in get_index(8.3.*)
    {"space_id":8,"type_id":3,"index[space_id].size":0}
    th_a  object_database.cpp:57 get_index
```

**Comment by mvandeberg** [comment]
No, it should not. The evaluator registry is only created for the follow plugin when the follow plugin is created. Can you please post a copy of the config file that generated this error so that I can reproduce the error?

**Comment by abitmore** [comment]
I noticed this once with a config file which is made by only removing account_history_plugin from default config file. The message shows when replay finished. Haven't tried to reproduce though.

**Comment by sneak** [comment]
Fixed in #424.
